<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pop_creation</class>
 <widget class="QWidget" name="pop_creation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Création du personnage</string>
  </property>
  <widget class="QLabel" name="labelCrea">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Création du personnage</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHabilete">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Habilete</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEndurance">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Endurance</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDisciplines">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelDisciplines">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Disciplines Kaï</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNotesDisciplines">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Notes discipline</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditDiscipline">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxMaitrise">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelMaitrise">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Maitrises Kaï</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAjouterDiscipline">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEnleverDiscipline">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Enlever</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelArmes">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Armes</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxHabilete">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxEndurance">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxArmes">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxInventaireArmes">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <property name="maxCount">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelArmes_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Inventaire d'armes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAjouterArme">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEnleverArme">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Enlever</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSacADos">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter au sac à dos</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxBourse">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelBourse">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Bourse</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelObjetsSpeciaux">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Objets spéciaux</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditObjetsSpeciaux">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNomPersonnage">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelNomPersonnage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom du personnage</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDemarrer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Démarrer!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAjouterSac">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEnleverSac">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Enlever</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditObjet">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxInventaire">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxCount">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelSac">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sac à dos</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
